src/components/EventDetails.tsx

```diff
@@ -0,0 +1,84 @@
+import { ReactElement } from "react";
+import Frame from ".././assets/Frame.svg";
+import LinkSVG from ".././assets/Link.svg";
+import MeatballMenuSVG from ".././assets/MeatballMenu.svg";
+import PinSVG from ".././assets/Pin.svg";
+import TextIconSVG from ".././assets/TextIcon.svg";
+import { Event } from "./EventCard";
+
+export type EventDetailsProps = {
+  event: Event;
+};
+
+const EventDetails = ( {event}:EventDetailsProps): ReactElement => {
+  return (
+    <div
+      className="flex h-screen w-screen flex-col p-10"
```

Comment:
Rather than doing `h-screen w-screen`, I'd recommend doing:
```suggestion
      className="flex flex-col flex-1 p-10"
```
As the whole page here is essentially a flexbox, and doing this means you want this `div` to take up the remaining space of the flexbox. I think what you have here is fine but this gives the `div` more flexibility in size